pwa/app/components/tba/navigation/footer.tsx

```diff
@@ -0,0 +1,89 @@
+import { Link } from '@tanstack/react-router';
+import { Fragment } from 'react/jsx-runtime';
+
+const links = [
```

Comment:
Tanstack router's typesafety is broken here. e.g. if you misspell `/about` to `/abot` it doesn't show a typechecking error.

Something like this could work:

```
import type { FileRouteTypes } from '~/routeTree.gen';

interface InternalLink {
  label: string;
  to: FileRouteTypes['fullPaths'];
}

interface ExternalLink {
  label: string;
  href: string;
}

type NavigationLink = InternalLink | ExternalLink;

const links: NavigationLink[] = [...
```